src/components/PathwaysService/PathwaysService.tsx

```diff
@@ -15,14 +15,21 @@ function typedFetch<T>(url: string, options?: object): Promise<T> {
 }
 
 const useGetPathwaysService = (url: string): Service<Pathway[]> => {
-  const [result, setResult] = useState<Service<Array<Pathway>>>({
+  const [result, setResult] = useState<Service<Pathway[]>>({
     status: 'loading'
   });
 
   useEffect(() => {
     getPathways(url)
-      .then(response => response.json() as Promise<Array<string>>)
-      .then(listOfFiles => listOfFiles.map(f => typedFetch<Pathway>(url + '/' + f)))
+      .then(response => response.json() as Promise<string[]>)
+      .then(listOfFiles =>
+        listOfFiles.map(f =>
+          typedFetch<Pathway>(url + '/' + f).then(pathway => ({
+            id: f,
```

Comment:
Instead of adding the id on as an after thought we should probably just add it to the pathway files. When we export the pathway it strips the ids and keys added but we will probably want to save the pathway with this, especially since pathways could probably be loaded from multiple directories meaning filenames are not unique